fast64_internal/render_settings.py

```diff
@@ -191,30 +184,62 @@ class Fast64RenderSettings_Properties(bpy.types.PropertyGroup):
         default=(0.5, 0.5, 0.5, 1),
         update=on_update_render_preview_nodes,
     )
-    lightColor: bpy.props.FloatVectorProperty(
-        name="Light Color",
+    light0Color: bpy.props.FloatVectorProperty(
+        name="Light 0 Color",
         subtype="COLOR",
         size=4,
         min=0,
         max=1,
         default=(1, 1, 1, 1),
         update=on_update_render_preview_nodes,
     )
-    lightDirection: bpy.props.FloatVectorProperty(
-        name="Light Direction",
+    light0Direction: bpy.props.FloatVectorProperty(
+        name="Light 0 Direction",
         subtype="DIRECTION",
         size=3,
         min=-1,
         max=1,
-        default=mathutils.Vector((0.5, 0.5, 1)).normalized(),  # pre normalized
+        default=mathutils.Vector((1.0, -1.0, 1.0)).normalized(),  # pre normalized
+        update=on_update_render_preview_nodes,
+    )
+    light0SpecSize: bpy.props.IntProperty(
+        name="Light 0 Specular Size",
+        min=1,
+        max=255,
+        default=3,
+        update=on_update_render_preview_nodes,
+    )
+    light1Color: bpy.props.FloatVectorProperty(
+        name="Light 1 Color",
+        subtype="COLOR",
+        size=4,
+        min=0,
+        max=1,
+        default=(0, 0, 1, 1),
+        update=on_update_render_preview_nodes,
+    )
+    light1Direction: bpy.props.FloatVectorProperty(
+        name="Light 1 Direction",
+        subtype="DIRECTION",
+        size=3,
+        min=-1,
+        max=1,
+        default=mathutils.Vector((-1.0, 1.0, -1.0)).normalized(),  # pre normalized
+        update=on_update_render_preview_nodes,
+    )
+    light1SpecSize: bpy.props.IntProperty(
+        name="Light 1 Specular Size",
+        min=1,
+        max=255,
+        default=3,
         update=on_update_render_preview_nodes,
     )
```

Comment:
should these be snake case?

more importantly maybe you should add upgrade code
```
def upgrade_changed_props():
        for scene in bpy.data.scenes:
            upgrade_old_prop(self, "light0Color", self, "lightColor")
            upgrade_old_prop(self, "light0Direction", self, "lightDirection"
)
```

Reply:
I have added some upgrade code and checked that it works.

As far as the names, since all the other properties I didn't change in this object are camelCase, I think it's best to keep the new names consistent with the others.